README.md

```diff
@@ -438,6 +442,77 @@ gsuite:
       months: 0
 ```
 
+### Pagerduty Config
+
+pytest-services does not query the pagerduty API, but can run tests against output from it.
+
+The config looks like:
+```
+pagerduty:
+  users_with_remote_access_monitoring: 'pd_users.json'
+  bastion_users: 'hierahash/*hierahash.json'
+  alternate_usernames: 'alternate_usernames.json'
+```
+
+Where `users_with_remote_access_monitoring` and `bastion_users` are
+globs for multiple files relative to the current working directory and
+`alternate_usernames` is the path to a single file.
+
+The files have examples formats as follows:
+
+* `users_with_remote_access_monitoring`:
+
+```json
+[
+  {
+    "avatar_url": "https://secure.gravatar.com/avatar/...",
+    "billed": true,
+    "color": "sea-green",
+    "contact_methods": [],
+    "description": null,
+    "email": "edunham@mozilla.com",
+    "html_url": "https://example-org.pagerduty.com/users/AAA0999",
+    "id": "AAA0999",
+    "invitation_sent": false,
+    "job_title": null,
+    "name": "C. Hobbes",
+    "notification_rules": [],
+    "role": "user",
+    "self_": "https://api.pagerduty.com/users/AAA0999",
+    "summary": "C. Hobbes",
+    "teams": [],
+    "time_zone": "America/New_York",
+    "type": "user"
+  },
+  ...
+]
+```
+
+* `bastion_users`:
+
+```json
+{
+    "chobbes": {
+        "groups": [
+            ""
+        ],
+        "root_ssh": true
+	},
+	"movedon": {
+        "ensure": "absent"
```

Comment:
nit: is this moving a closing `}`?

Reply:
Yeah, the whitespace is off here. I'll clean that up.

Reply:
mixed tabs and spaces :scream: